### PR TITLE
[user_status] Improve user online status modal

### LINKED ISSUE #37097: [BITV] 9.4.1.2/2.5 - "Online status" Modal Dialog - The status buttons such as "In a meeting" have not been implemented as buttons using HTML button element or ARIA role="button," but rather as focusable div elements. Instead of buttons, one could use radio buttons, for example. (1)
- [ ] Fix
- [ ] Backport

<details>
<summary>Details</summary>

https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-bfbf07e5c3-v2-n5

</details>

### PR DESCRIPTION
**Pull request by susnux**
* Resolves: #37097

## Summary
1. Fix the predefined statuses buttons to implement a role (radio)¹ instead of being just tabable divs
    * Also set role to radiogroup of list container to group the predefined statuses
    * ¹Use native radio buttons 
 2. Group custom message input elements and assign label to emoji selector
 3. Add missing `radiogroup` role to online status selector

## TODO
<del>For 1: I have also created a commit where I use real radio buttons (in the background) for the predefined statuses, so the question is should I push it here too? Do we want native radio buttons instead? What do you think @Pytal @JuliaKirschenheuter ? </del>

## Checklist

- Code is [properly formatted](https://docs.nextcloud.com/server/latest/developer_manual/digging_deeper/continuous_integration.html#linting)
- [Sign-off message](https://github.com/src-d/guide/blob/master/developer-community/fix-DCO.md) is added to all commits
- [x] Screenshots before/after for front-end changes **no visual changes**
- [ ] [Backports requested](https://github.com/nextcloud/backportbot/#usage) where applicable (ex: critical bugfixes)

### CODE REVIEW

**Comment by Pytal** [comment]
Let's go with real radio inputs @susnux as it's the recommended way https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/radio_role#sect1 :)

**Review by Pytal — APPROVED** [review]

**Comment by nickvergessen** [comment]
Can we have before and after screenshots?

**Review by Antreesy — APPROVED** [review]
image

No visual changes (before => after), except for one:
* when selected 'button' is not in focus, it now keep selected styles

No blocking changes, except for redundant class

**Review by Antreesy — CHANGES_REQUESTED** [review]
And please rebuild

File: apps/user_status/src/components/PredefinedStatus.vue [hunk 1]
```diff
@@ -76,39 +90,52 @@ export default {
...
-		}
+	// Inputs are here for keyboard navigation, they are not visually visible
+	&__input {
+		position: absolute;
+		top: auto;
+		left: -10000px;
+		overflow: hidden;
+		width: 1px;
+		height: 1px;
 	}
```
[the remark this answers is not in the document]
Comment (susnux) [reply apps/user_status/src/components/PredefinedStatus.vue:139]: We should then use it also for `OnlineStatusSelect.vue`

**Review by Antreesy — APPROVED** [review]
:seal:

**Comment by susnux** [comment]
rebased

**Pull request merged**